### PR TITLE
Update cli-using-daemon-mode.md

### PR DESCRIPTION
**Pull request by aimmta**
Signed-off-by: Michael Aimino <michael.aimino@broadcom.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
- [x] Cleanup documentation that may be referencing "zowex"

- [x] Cleanup documentation that may be referencing daemon startup commands

- [ ] Publish new @next version of Zowe CLI to public npm and binary executables to GitHub

https://rally1.rallydev.com/#/227079578160ud/teamplan?detail=%2Fuserstory%2F613392093263

### CODE REVIEW

**Review by gejohnston — CHANGES_REQUESTED** [review]
The behavior of the zowe executable has been changed, so the section titled "Starting Zowe in daemon mode" must be rewritten. I suggest the following:

Running zowe commands with the zowe executable

Verify that you have installed the @next release of Zowe CLI and the zowe native executable.

Run any zowe command as you normally would. The first time you run any zowe command, the command will automatically start a daemon in the background. It will then run your desired command. Since that first command must start the daemon, that first zowe command will actually run slower than a traditional zowe command. However, every zowe command afterward will run significantly faster. The daemon will continue to run in the background until you close your terminal window.

Example:
```
    zowe --version
    Starting a background process to increase performance ...
    7.0.0-next.202110211759

    zowe --version
    7.0.0-next.202110211759
```

File: docs/user-guide/cli-using-daemon-mode.md [hunk 1]
```diff
@@ -5,45 +5,40 @@
...
+To download and install the zowe native executable from the Zowe Github repository:
 
-To download and install the zowex native executable from the Zowe Github repository:
 1. Navigate to the Zowe CLI GitHub [repository](https://github.com/zowe/zowe-cli)
 2. Under the **Release** section, click **Native Client Release**.
    The Download page is displayed.
 3. Click the link to download the executable (.tgz file) for your operating system.
-4. Unzip the zowex .tgz file and place the zowex (or zowex.exe) file into a directory which occurs on your PATH earlier than the directory containing the NodeJS zowe command.
-
+4. Unzip the zowe .tgz file and place the zowe (or zowe.exe) file into a directory which occurs on your PATH earlier than the directory containing the NodeJS zowe command.
```
Comment (zFernand0) [inline docs/user-guide/cli-using-daemon-mode.md:27]: I'd like to add a note (for Mac Users) alongside these steps to indicate that people may (or may not) need to accept GitHub as an unidentified developer.
More details (and link with steps) can be found in the issue:
- https://github.com/zowe/zowe-cli/issues/1193

**Review by zFernand0 — COMMENTED** [review]
Added some comments in regards to the unidentified GitHub developer warning for Mac Users 😋

File: docs/user-guide/cli-using-daemon-mode.md [hunk 2]
```diff
@@ -5,45 +5,40 @@
...
 
 ## Feature overview
+
 Daemon Mode significantly improves the performance of Zowe CLI commands by running Zowe CLI as a persistent background process (daemon).
 
-## Install the zowex native executable
-You can download and install the zowex native executable from Zowe Downloads (preferred) or from the Zowe Github repository. To install the zowex native executable from Zowe Downloads, see [Install Zowe CLI from a download](cli-install-cli-next.md#install-zowe-cli-from-a-download).
+## Install the zowe native executable
+
+You can download and install the zowe native executable from Zowe Downloads (preferred) or from the Zowe Github repository. To install the zowe native executable from Zowe Downloads, see [Install Zowe CLI from a download](cli-install-cli-next.md#install-zowe-cli-from-a-download).
```
[the remark this answers is not in the document]
Comment (aimmta) [reply docs/user-guide/cli-using-daemon-mode.md:19]: Should I hold off on making further changes until these issues are completed?

[the remark this answers is not in the document]
Comment (zFernand0) [reply docs/user-guide/cli-using-daemon-mode.md:19]: For now, we can hold off on documentation updates related to the `zowe config daemon` command until the two issues above are completed.
However, we should still add the Mac-only note regarding the unidentified developer issue (next comment - https://github.com/zowe/docs-site/pull/1882#discussion_r742204064)

**Review by gejohnston — APPROVED** [review]
While I believe that Fernando has some additional updates, I approve the changes that I requested.

**Comment by aimmta** [comment]
Added the note for Mac users. Going to merge this PR and address the changes for zowe/zowe-cli#1175
zowe/zowe-cli#1176 when those are ready.

**Pull request merged**